src-tauri/src/app_config.rs

```diff
@@ -12,16 +12,24 @@ const LOG_TARGET: &str = "tari::universe::app_config";
 #[derive(Debug, Clone, Serialize, Deserialize)]
 #[allow(clippy::struct_excessive_bools)]
 pub struct AppConfigFromFile {
-    pub version: u32,
-    pub mode: String,
-    pub auto_mining: bool,
-    pub p2pool_enabled: bool,
-    pub last_binaries_update_timestamp: SystemTime,
-    pub allow_telemetry: bool,
-    pub anon_id: String,
-    pub monero_address: String,
-    pub gpu_mining_enabled: bool,
-    pub cpu_mining_enabled: bool,
+    #[serde(default = "default_mode")]
+    mode: String,
+    #[serde(default = "default_false")]
+    auto_mining: bool,
+    #[serde(default = "default_false")]
+    p2pool_enabled: bool,
+    #[serde(default = "default_system_time")]
+    last_binaries_update_timestamp: SystemTime,
+    #[serde(default = "default_true")]
```

Comment:
```suggestion
    #[serde(default = "default_false")]
```
This must be false unless they have accepted